SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -149,6 +151,24 @@ private void submitRequest(SingularityRequest request, Optional<SingularityReque
       requestState = oldRequestWithState.get().getState();
     }
 
+    if (oldRequest.isPresent() && oldRequest.get().getInstancesSafe() < request.getInstancesSafe()) {
```

Comment:
I think we want to swap the operands on this comparison.

Reply:
We do indeed, hadn’t actually gotten a chance to finish unit test and testing in staging in this one yet